src/spacel/provision/app/ingress_resource.py

```diff
@@ -102,3 +111,19 @@ def app_sg(self, orbit, region, app, sg_ref='Sg'):
                              orbit.name, region)
                 return None
             raise e
+
+    def app_eips(self, orbit, region, app, eips=[]):
```

Comment:
`eips=[]`: mutable arguments are evil.

I think if you run this function twice, the second execution will return duplicate elements.